text/exercise1/cake.md

```diff
@@ -30,4 +35,3 @@ Stir in vanilla and pour cake batter into the prepared pan.
 
 Bake in the preheated oven until a toothpick inserted in the center comes out clean, about 40 to 45 minutes. 
```

Comment:
I think there's a step or two missing here that combines the wet and dry ingredients and pours it into the container.

Suggestion:

Mix the flour mixture into the wet mixture until combined.
Pour into prepared baking dish.